900.version-fixes/g.yaml

```diff
@@ -144,95 +141,46 @@
-- { name: gnome-devel-docs,            verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-disk-utility,          verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
 - { name: gnome-documents,             verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
 - { name: gnome-flashback,             verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-font-viewer,           verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-games,                 verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-games-app,             verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-getting-started-docs,  verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
 - { name: gnome-icon-theme,            ver: "3.14.0",                ruleset: pclinuxos,   incorrect: true }
 - { name: gnome-icon-theme,                                          ruleset: pclinuxos,   untrusted: true } # accused of fake 3.14.0
-- { name: gnome-initial-setup,         verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
 - { name: gnome-js-common,             verpat: "20[0-9]{6}",                               snapshot: true }
-- { name: gnome-keyring,               verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
 - { name: gnome-klotski,               verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
 - { name: gnome-latex,                 verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-logs,                  verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
 - { name: gnome-mahjongg,              verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-maps,                  verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
 - { name: gnome-menus,                 verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-mines,                 verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
 - { name: gnome-mplayer,               verpat: ".*[ab]",                                   devel: true }
 - { name: gnome-multi-writer,          verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
 - { name: gnome-nibbles,               verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-notes,                 verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
 - { name: gnome-online-accounts,       verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
 - { name: gnome-online-miners,         verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
-- { name: gnome-packagekit,            verpat: "[0-9]+\\.[0-9]*[13579]\\..*",              devel: true }
```

Comment:
This too

Reply:
Hm, I thought packagekit was totally dead. No releases since 3.32.0 3 years ago (https://gitlab.gnome.org/GNOME/gnome-packagekit/-/tags) but it's not officially archived.

I'll restore this rule.